### ISSUE TITLE
[Bug] bench_serving.py has a issue.

### ISSUE THREAD

**Issue by aLowMagic** [sgl-project/sglang#1075]
### Checklist

- [X] 1. I have searched related issues but cannot get the expected help.
- [X] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.
- [x] 4. If the issue you raised is not a bug but a question, please raise a discussion at https://github.com/sgl-project/sglang/discussions/new/choose Otherwise, it will be closed.

### Describe the bug

Cannot run the bench_serving successfully for some reason. (bench_one.py works good)

### Reproduction

```
/sgl-workspace/sglang/benchmark/latency_throughput# python3 bench_serving.py --host 0.0.0.0 --port 30000 --backend srt --tokenizer meta-llama/Meta-Llama-3-8B-Instruct --num-prompt 1 --request-rate 100
Namespace(backend='srt', host='0.0.0.0', port=30000, dataset=None, input_len=2048, output_len=256, range_ratio=1.0, tokenizer='meta-llama/Meta-Llama-3-8B-Instruct', best_of=1, use_beam_search=False, num_prompts=1, request_rate=100.0, seed=0, trust_remote_code=False)
  0%|                                                                                                                                                              | 0/1 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/sgl-workspace/sglang/benchmark/latency_throughput/bench_serving.py", line 376, in <module>
    main(args)
  File "/sgl-workspace/sglang/benchmark/latency_throughput/bench_serving.py", line 288, in main
    asyncio.run(
  File "/usr/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/sgl-workspace/sglang/benchmark/latency_throughput/bench_serving.py", line 245, in benchmark
    await tqdm_asyncio.gather(*tasks)
  File "/usr/local/lib/python3.10/dist-packages/tqdm/asyncio.py", line 79, in gather
    res = [await f for f in cls.as_completed(ifs, loop=loop, timeout=timeout,
  File "/usr/local/lib/python3.10/dist-packages/tqdm/asyncio.py", line 79, in <listcomp>
    res = [await f for f in cls.as_completed(ifs, loop=loop, timeout=timeout,
  File "/usr/lib/python3.10/asyncio/tasks.py", line 571, in _wait_for_one
    return f.result()  # May raise f.exception().
  File "/usr/local/lib/python3.10/dist-packages/tqdm/asyncio.py", line 76, in wrap_awaitable
    return i, await f
  File "/sgl-workspace/sglang/benchmark/latency_throughput/bench_serving.py", line 181, in send_request
    async with session.post(
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/client.py", line 1354, in __aenter__
    self._resp = await self._coro
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/client.py", line 587, in _request
    raise err_exc_cls(url)
aiohttp.client_exceptions.InvalidUrlClientError: 0.0.0.0:30000/generate
  0%|                                                                                                                                                              | 0/1 [00:00<?, ?it/s]
```

### Environment

```
root@SC-FAT-EHPC-SPR8480-805:/sgl-workspace/sglang/benchmark/latency_throughput# python3 -m sglang.check_env
Python: 3.10.14 (main, Apr  6 2024, 18:45:05) [GCC 9.4.0]
CUDA available: False
PyTorch: 2.4.0+cu121
sglang: 0.2.12
flashinfer: 0.1.4+cu121torch2.4
triton: 3.0.0
transformers: 4.44.0
requests: 2.32.3
tqdm: 4.66.5
numpy: 1.26.4
aiohttp: 3.10.3
fastapi: 0.112.0
hf_transfer: 0.1.8
huggingface_hub: 0.24.5
interegular: 0.3.3
packaging: 24.1
PIL: 10.4.0
psutil: 6.0.0
pydantic: 2.8.2
uvicorn: 0.30.5
uvloop: 0.19.0
zmq: 26.1.0
vllm: 0.5.4
multipart: 0.0.9
openai: 1.40.6
anthropic: 0.33.1
ulimit soft: 1048576

```

**Comment by hnyls2002** [comment]
@aLowMagic For the bench_serving script, you have to launch a server first.

**Comment by aLowMagic** [comment]
Hello, @hnyls2002. Thank you for the reply!
Actually I did. And I can get log when I'm running bench_one.py so it seems not because of the server didn't started:
```
python3 bench_one.py --input-len 128 --port 30000
[{'text': ' 0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0', 'meta_info': {'prompt_tokens': 128, 'completion_tokens': 256, 'completion_tokens_wo_jump_forward': 256, 'finish_reason': 'FINISH_LENGTH: 256', 'id': '2612dfb9b1b74ab981f3032e1e7224a9'}}]
latency: 5.61 s
output throughput: 45.61 token/s
(input + output) throughput: 68.41 token/s

```
And can get log from the server:
```
INFO:     127.0.0.1:39020 - "POST /generate HTTP/1.1" 200 OK
The server is fired up and ready to roll!
[gpu=0] Prefill batch. #new-seq: 1, #new-token: 128, #cached-token: 0, cache hit rate: 0.00%, #running-req: 0, #queue-req: 0
......
[gpu=0] Decode batch. #running-req: 1, #token: 361, token usage: 0.00, gen throughput (token/s): 45.69, #queue-req: 0
```
But still got the error when I'm running bench_serving.py:
```
python3 bench_serving.py --port 30000 --backend srt --host 0.0.0.0 --input-len 128 --output-len 128 --num-prompts 1 --tokenizer meta-llama/Meta-Llama-3-8B --request-rate 1 --trust-remote-code
Namespace(backend='srt', host='0.0.0.0', port=30000, dataset=None, input_len=128, output_len=128, range_ratio=1.0, tokenizer='meta-llama/Meta-Llama-3-8B', best_of=1, use_beam_search=False, num_prompts=1, request_rate=1.0, seed=0, trust_remote_code=True)
  0%|                                                                                                                                                                 | 0/1 [00:00<?, ?it/s]
......
  File "/usr/local/lib/python3.10/dist-packages/tqdm/asyncio.py", line 76, in wrap_awaitable
    return i, await f
  File "/sgl-workspace/sglang/benchmark/latency_throughput/bench_serving.py", line 180, in send_request
    async with session.post(
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/client.py", line 1353, in __aenter__
    self._resp = await self._coro
  File "/usr/local/lib/python3.10/dist-packages/aiohttp/client.py", line 520, in _request
    raise NonHttpUrlClientError(url)
aiohttp.client_exceptions.NonHttpUrlClientError: 0.0.0.0:30000/generate
  0%|                                                                                                                                                                 | 0/1 [00:00<?, ?it/s]
```

Thank you!

**Comment by aLowMagic** [comment]
Fixed by `--host http://0.0.0.0`
